torchci/components/GroupJobConclusion.tsx

```diff
@@ -8,14 +8,15 @@ import {
   PinnedTooltipContext,
 } from "pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]]";
 
-enum JobStatus {
+export enum JobStatus {
```

Comment:
This changes the semantics of `JobStatus`, it no longer refers to "the status of a CI job" but "the status of a CI job OR the aggregate status of a job". That's a bit messy, since I would expect all values of `JobStatus` to be valid job statuses, but that's no longer the case (a job can't have the status `AllSkipped`).

If we want to use this `JobStatus` enum as a basic primitive throughout the codebase (which seems like a good idea and this PR goes in that direction a bit), it's probably worth making the abstraction more accurate. Probably by splitting it into `JobStatus` and `GroupedJobStatus`, where `GroupedJobStatus` derives from `JobStatus`?